gocd/bci.gocd.yaml

```diff
@@ -106,23 +106,25 @@ pipelines:
         - SLE
         environment_variables:
           OSC_CONFIG: /home/go/config/oscrc-totest-manager
+          BCI_TOKEN: '{{SECRET:[opensuse.secrets][BCI_TOKEN]}}'
         resources:
         - staging-bot
         tasks:
         - script: |-
+            SPRJ=SUSE:SLE-15-SP3:Update:BCI
             for arch in aarch64 ppc64le s390x x86_64 ; do
-             SPRJ=SUSE:SLE-15-SP3:Update:BCI
-             PKG="000product:SLE_BCI-ftp-POOL-$arch"
-             PRJ="SUSE:Products:SLE-BCI:15-SP3:$arch"
-             osc -A https://api.suse.de api -X POST "/source/$SPRJ/$PKG?cmd=release&target_project=$PRJ&repository=images&target_repository=images"
+              PKG="000product:SLE_BCI-ftp-POOL-$arch"
+              PRJ="SUSE:Products:SLE-BCI:15-SP3:$arch"
+              curl -X POST -H "Authorization: Token $BCI_TOKEN" "https://api.suse.de/trigger/release?project=$SPRJ\&package=000product:SLE_BCI-ftp-POOL-$arch\&targetproject=$PRJ\&targetrepository=images\&filter_source_repository=\"images\""
+              #              osc -A https://api.suse.de/ token --trigger $BCI_TOKEN --operation release $SPRJ $PKG
```

Comment:
please reference https://github.com/openSUSE/osc/issues/1194 and drop the commented code.